pytest_fixtures/component/provision_pxe.py

```diff
@@ -289,6 +311,14 @@ def provisioning_hostgroup(
             },
         ],
     ).create()
+    # subnet = module_provisioning_sat.subnet
+    if settings.server.is_ipv6:
+        hostgroup.subnet6 = module_provisioning_sat.subnet
+        hostgroup.update(['subnet6'])
+    else:
+        hostgroup.subnet = module_provisioning_sat.subnet
+        hostgroup.update(['subnet'])
+    return hostgroup
```

Comment:
Can we set this during hostgroup creation itself with something like?
```
subnet= module_provisioning_sat.subnet if not settings.server.is_ipv6 else None
subnet6=module_provisioning_sat.subnet if settings.server.is_ipv6 else None
```

Reply:
Updated